extensions/objects/templates/number-with-units-editor.component.ts

```diff
@@ -47,10 +47,13 @@ export class NumberWithUnitsEditorComponent implements OnInit {
   }
 
   ngOnInit(): void {
-    if (this.value !== null) {
+    if (this.value === null || this.value === undefined) {
+      return;
+    } else {
       const defaultNumberWithUnits =
-        this.numberWithUnitsObjectFactory.fromDict(this.value);
-      this.numberWithUnitsString = defaultNumberWithUnits.toString();
+        this.numberWithUnitsObjectFactory.fromDict(
+          this.value as NumberWithUnitsAnswer);
+      this.numberWithUnitsString = defaultNumberWithUnits?.toString();
```

Comment:
Why `?`?

Reply:
this file is migrated by @srijanreddy98 and sometimes there was a flake in the e2e test that why I have used ? here

Reply:
removed in PR https://github.com/oppia/oppia/pull/15859, PTAL